cernopendata/modules/theme/assets/semantic-ui/js/glossary/jquery.zglossary.js

```diff
@@ -27,13 +27,13 @@ import jquery from "jquery/dist/jquery";
 				var skip = 0;
 
 				// Check the element is a text node
-				if (e.nodeType == 3) {
+				if (e.nodeType == 3 && e.textContent.trim().length) {
 
 					// Case insensistive matching option
 					if (options.ignorecase) {
-						var pos = e.data.toLowerCase().search(new RegExp('\\b'+patfmt.toLowerCase()+'\\b'));
+						var pos = e.data.toLowerCase().search(new RegExp('\\b'+patfmt.toLowerCase()+'\\b(?!^\)'));
 					} else {
-						var pos = e.data.search(new RegExp('\\b'+patfmt+'\\b'));
+						var pos = e.data.search(new RegExp('\\b'+patfmt+'\\b(?!^\)'));
```

Comment:
Helps for the example record cited in the issue, but for another record (see below) I have seen the following with this PR:

![charged](https://user-images.githubusercontent.com/517546/107205791-8641c900-69fe-11eb-9e64-164dff41da86.png)

for the record http://localhost:5000/record/10107 coming from `cernopendata/modules/fixtures/data/records/opera-detector-events-tau.json`.

Can you please have a look at refining the regexp some more?

Reply:
BTW do you also see this "overemphasised TeX style" when compared to the style of the main text?  I'm seeing it also with Chromium, but it is less pronounced there. The letter sizes are more comparable, but seem to end up "bolder" than the main text. (Note how In the middle of reload, the "2nd" part is handled by Mathjax and it looks OK, and then it is only at the end of the reload when it turns "bolder"):

![record-10107-chromium](https://user-images.githubusercontent.com/517546/107209266-dde23380-6a02-11eb-9c1f-36f20cbab669.png)

Reply:
With https://github.com/cernopendata/opendata.cern.ch/pull/3047/commits/6bc14993d76c539475a7dfd84bad9ca1800cb1f8 the font doesn't get bigger, but still there is a second font size change... let me know how it looks for you. Otherwise we can merge without this commit and tackle this problem separately since apparently, it has to do with MathJax only.